composer.json

```diff
@@ -101,10 +101,9 @@
         "zendframework/zendsearch": "@dev"
     },
     "conflict": {
-        "symfony/symfony": "2.8.10",
-        "symfony/phpunit-bridge": "2.8.10",
-        "symfony/monolog-bundle": "2.8.10",
+        "doctrine/lexer": "1.0.0",
```

Comment:
strange problem with composer with doctrine/orm and symfony/swiftmailer-bundle https://github.com/composer/composer/issues/7161